packages/database/src/exp/Database.ts

```diff
@@ -51,7 +51,7 @@ import { validateUrl } from '../core/util/validation';
 import { ReferenceImpl } from './Reference_impl';
 
 /**
- * This variable is also defined in the firebase node.js admin SDK. Before
+ * This variable is also defined in the firebase NodeJS Admin SDK. Before
```

Comment:
We use "Node.js" in the docs. See e.g. https://firebase.google.com/docs/admin/setup.

Reply:
Changed.